src/learning_item_data_retriever.py

```diff
@@ -0,0 +1,75 @@
+import json
+import pandas as pd
+from learning_item import LearningItem
+
+FOLDER_PATH = "E:\\Mi unidad\\_PYTHON\\PycharmProjects\\memoryBooster\\data\\"
+SOURCE_FILE = "data_esp.txt"
+RESULT_FILE = "data_esp2.json"
+
+def learning_items_from_file(file_path):
+    with open(file_path, 'r', encoding='utf-8') as file:
+        lines = file.readlines()
+
+    learning_items = [ ]
+    number_of_units = int(lines[0])
+    num_lines = len(lines)
+    i = 2
+    while (i < num_lines):
+        if lines[i].strip() == '-' * len(lines[i].strip()):
+            i += 1
+            continue
+        id = int(lines[i].strip())
+        isTaught = lines[i + 1].strip() == "true"
+        isVerbIrregular = lines[i + 2].strip() == "true"
+        questions = []
+        for j in range(3, 6):
```

Comment:
Also there are two parts of code that look almost identical only the range changes. Extract this as well and use it in the method for parsing learning item